src/prometheus/prometheus-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Isn't this file redundant? The info is already in README file.

Reply:
I wanted to describe some sections of Prometheus deployment in the markdown file and also I wanted to have the possibility to run `kubectl apply` and `kubectl delete` without copy-pasting.

Reply:
Than it should be included in README.md from here or just tangled from README. Having same file on multiple places is hard to maintain.

Reply:
fixed